README.md

```diff
@@ -10,9 +10,17 @@ The **bipedal-locomotion-controllers** project is a _suite_ of libraries for ach
 - [:running: How to use the libraries](#running-how-to-use-the-libraries)
 
 # :orange_book: Exported components
+- [**OptimalControlUtilities**](./src/OptimalControlUtilities): Utilities library for handling cost and
+  constraints in a quadratic optimization problem (QP)
 
 # :page_facing_up: Dependencies
+The **bipedal-locomotion-controllers** project is versatile and it can be used
+  to compile only some components. Each component has its own dependencies that
+  can be found in [`BipedalLocomotionControllersFindDependencies.cmake`](./cmake/BipedalLocomotionControllersFindDependencies.cmake)
+  file. Please note that the indicated version is the minimum required version.
 
+- `OptimalControlUtilities` requires:
+    - [`iDynTree`](https://github.com/robotology/idyntree) (version 0.11.105)
```

Comment:
Note to self: I need to release iDynTree 0.12 before the end of the year.

Reply:
I guess this has been done :grinning: